docs/developer-docs/contributing.md

```diff
@@ -1,34 +1,34 @@
 # Contributing to Jellyfin
 
 This page details how you can help contribute to Jellyfin. Please give this a read-through before opening an Issue or PR, or if you're not sure how to do either.
-                                                                                              
+
 Are you a coder and interested in [writing and maintaining C# or Javascript code](/developer-docs/contributing#contributing-code-to-jellyfin)? Or are you not a coder and interested in [reporting an issue or contributing in other ways](/developer-docs/contributing#other-contributions-to-jellyfin)?
-                                                           
+
 ## Contributing Code to Jellyfin        
 
 The Jellyfin project consists of a C# core server, a Javascript-based UI, and a number of client applications written in various languages and frameworks. If you have experience with these languages, we're always grateful for any contributions you might want to make!
 
 ### What should you do?
-     
+
 The best way to get going is to look through the [Issues list](https://github.com/jellyfin/jellyfin/issues), find an issue you would like to work on, and start hacking. Issues are triaged regularly by the administrative team, and labels assigned that should help you find issues within your skill-set. Once you start working on an issue, please comment on it stating your intent to work on the issue, to avoid unnecessary duplication of work.
-                                                                                                   
+
 #### Major Issue Types
 
 A list of issue types can be found on the [Issue guidelines](/developer-docs/issues#issue-labels) page.
 
 #### What if there isn't an issue?
 
 If there isn't already an issue dealing with the changes you want to make, please [create an issue](/developer-docs/issues) to track it first, then ensure your PR(s) reference the issue in question. This is epecially useful for bugs that are found and then fixed by the author, so both the original issue and the fix can be documented and tracked in detail.
-                                                                                        
+
 ### How should you make changes?
-                                 
+
 Once you've found something you want to work on or improve, the next step is to make your changes in the code, test them, then submit a PR. This section details how to go about doing that. For simplicity, all examples assume the developer is operating on Linux with SSH access to GitHub, however the general ideas can be translated to Windows or MacOS.
 
 If you aren't familiar with Git, we recommend the [official documentation](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control) to get yourself going.
 
-#### Set up your copy of the repo
+#### Set up your copy of the repo and make changes
```

Comment:
I don't like splitting these sections as it makes it harder to follow IMO. The detailed instructions help anyone who isn't well-versed in Git development follow the methodology.